CDL-Dryad/stash
# Large file fees

Detection and handling of large file fees. (https://github.com/CDL-Dryad/dryad-product-roadmap/issues/198)

Note that this requires https://github.com/cdlib/dryad-config/pull/58

To test: 
- Create a new dataset, and include a file that is larger than the `large_file_size` set in the config
- On the review page, verify that there is a warning message about large file fees
- Once the dataset is in `submitted` status, an email should be sent to `helpdesk_email`, which now defaults to `dryad.helpdesk@mailinator.com` for non-production servers
